README.md

```diff
@@ -76,6 +125,20 @@ pitest {
 }
 ```
 
+<details>
+<summary>with Kotlin DSL</summary>
+
+```kotlin
+pitest {
+    targetClasses.set(setOf("our.base.package.*")) //by default "${project.group}.*"
+    pitestVersion.set("1.15.0") //not needed when a default PIT version should be used
+    threads.set(4)
+    outputFormats.set(setOf("XML", "HTML"))
+    timestampedReports.set(false)
```

Comment:
Btw, Gradle 8.1 "simple property assignment" also in Kotlin DSL and in 8.4 it was [marked as stable](https://docs.gradle.org/8.4/release-notes.html#simple-property-assignment-in-kotlin-dsl-is-now-stable). While the syntax above is more idiomatic (and more portable), maybe you could make some note after that (first) example, that for people using newer Gradle, it could be simplified (with en example how) - but just for that one case, people reading README could adjust also other configurations to the simpler syntax if wanted/needed. WDYT?

Reply:
Thank you very much! I missed new syntax.